### PR TITLE
Update Ewon Branding Capitalization

### PR DESCRIPTION
**Pull request by alexjhawk**
Update the Ewon branding to have proper capitalization

### CODE REVIEW

**Comment by TomKimsey** [comment]
It looks fine to me but im not going to approve until I test it and make sure everything works as it should.  It looks like your editor or some tool is adding newlines to the end of files.  Its slightly more proper to have newlines at the end of files but for 99% of things it doesn't matter.

**Comment by TomKimsey** [comment]
@alexjhawk  Can you add like 28 of EwonConnectorSettings.java to this.  Change the default from eWon to Ewon

**Comment by alexjhawk** [comment]
> @alexjhawk Can you add like 28 of EwonConnectorSettings.java to this. Change the default from eWon to Ewon

Yes, I'll add that and update the PR

**Comment by alexjhawk** [comment]
> @alexjhawk Can you add like 28 of EwonConnectorSettings.java to this. Change the default from eWon to Ewon

Fixed

File: eWonConnector-gateway/src/main/resources/ewon_gateway.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ewon.Name=eWon Connector
-config.menutitle=eWon Connector
-config.settingsmenu=Settings
\ No newline at end of file
+ewon.Name=Ewon Connector
+config.menutitle=Ewon Connector
+config.settingsmenu=Settings
```
Comment (TomKimsey) [inline eWonConnector-gateway/src/main/resources/ewon_gateway.properties]: Newline was added

File: tools/MakeRelease.py [hunk 2]
```diff
@@ -77,4 +77,4 @@ def CreateRelease():
       print "Usage: MakeRelease.py"
       sys.exit(0)
 
-   CreateRelease()
\ No newline at end of file
+   CreateRelease()
```
Comment (TomKimsey) [inline tools/MakeRelease.py]: Newline was added

File: eWonConnector-gateway/src/main/resources/org/imdc/ewon/config/EwonConnectorSettings.properties [hunk 3]
```diff
@@ -17,6 +17,6 @@ historyEnabled.Name=History Enabled
 historyProvider.Name=Target History Provider
 
 Category.Main=Main
-Category.Account=eWon Account Information
-Category.Device=eWon Device Information
-Category.History=History
\ No newline at end of file
+Category.Account=Ewon Account Information
+Category.Device=Ewon Device Information
+Category.History=History
```
Comment (TomKimsey) [inline eWonConnector-gateway/src/main/resources/org/imdc/ewon/config/EwonConnectorSettings.properties]: Newline was added

**Pull request merged**